php-exercise-1/Controller/helloController.php

```diff
@@ -0,0 +1,18 @@
+<?php
+include 'Model/helloModel.php';
```

Comment:
Please use namespace instead of including files directly.

Please refer to PSR-4 for autoloading your files.

Reply:
I have modified the code and added an autoloader in the index.php file